supply-chain-security/compromises/2021/travis-ci.md

```diff
@@ -0,0 +1,27 @@
+# Secrets leak and other sensitive information due to a bug in Travis CI 
+
+Travis CI is a popular continuous integration (CI) and build service,
+originally forked from the Jenkins project, and later spun-off as a paid
+service. On September 23rd, 2021, they reported [1] that users and customers
+of their service, who had sensitive information in public repositories, and
+of which these were forked, had leaked these secrets in the build
+infrastructure logs were the CI service was ran.
+
+
+## Impact
+
+* All public repositories that observed forks during the period of September 3rd
+and September 10th
+* Any environment variables that were used as protected secrets in the project's
+configuration were leaked and available to CI runs in publicly forked repositories.
+
+## Remediation
+
+* Travis CI notified users and customers to rotate their keys and secrets
+
+## Type of Compromise
+
+Environment variables.
```

Comment:
Compromise type should be one of https://github.com/cncf/tag-security/blob/main/supply-chain-security/compromises/compromise-definitions.md, and you can add free from text elaborating on the env variables

Reply:
resolved via commit https://github.com/cncf/tag-security/pull/800/commits/3b3a74330adbfc337b44cf8c80eba65ac7fbcffe